bibtex_print/bibtex_print.py

```diff
@@ -8,14 +8,22 @@
 
 
 def load_bibtex(bibfile):
+    def mo_co(mo):
+        MONTH_CONVERT = {'': 0, 'jan': 1, 'Jan': 1, 'January': 1, 'feb': 2, 'Feb': 2, 'February': 2, 'mar': 3, 'Mar': 3, 'March': 3, 'apr': 4, 'Apr': 4, 'April': 4, 'may': 5, 'May': 5, 'june': 6, 'June': 6, 'jul': 7, 'july': 7, 'July': 7,
+                         'aug': 8, 'Aug': 8, 'August': 8, 'sep': 9, 'Sep': 9, 'september': 9, 'September': 9, 'oct': 10, 'Oct': 10, 'october': 10, 'October': 10, 'nov': 11, 'Nov': 11, 'November': 11, 'dec': 12, 'Dec': 12, 'December': 12}
```

Comment:
you aren't consistent with the variations in capitalizations and abbreviations.

for instance, you check `dec`, `Dec`, and `December`, but not `december`

but for September you check all 4.  i'd just go through and check them all, though I think we could do this more succinctly with regex.